src/scripts/dbgap/dbgap_classes.py

```diff
@@ -500,54 +475,75 @@ def get_submission_comment():
+                    "TAG": "Submission Site",
+                    "VALUE": "NCBI_PROTECTED"
+                }
+            }
+        }
+        submission.update(submission_attributes)
 
     def create_file(self):
         print("Creating submission xml file")
 
-        root = ET.Element("SUBMISSION_SET")
-        root.set("xsi:noNamespaceSchemaLocation", SUBMISSION_XSD)
-        root.set("xmlns:xsi", XSI)
-        submission = ET.SubElement(root, "SUBMISSION", submission_date=get_date(), submission_comment=self.get_submission_comment(), lab_name="Genome Sequencing", alias=self.get_alias(), center_name=BROAD_ABBREVIATION)
-
-        contacts = ET.SubElement(submission, "CONTACTS")
-        contact = ET.SubElement(contacts, "CONTACT", name="sra_sumissions", inform_on_status="mailto:dsde-ops@broadinstitute.org", inform_on_error="mailto:dsde-ops@broadinstitute.org")
+        submission_dict = {
+            "SUBMISSION_SET": {
+                "@xsi:noNamespaceSchemaLocation": SUBMISSION_XSD,
+                "@xmlns:xsi": XSI,
+                "SUBMISSION": {
+                    "@submission_date": get_date(),
+                    "@submission_comment": self.get_submission_comment(),
+                    "@lab_name": "Genome Sequencing",
+                    "@alias": self.get_alias(),
+                    "@center_name": BROAD_ABBREVIATION,
+                    "CONTACTS": {
+                        "CONTACT": {
+                            "@name": "sra_sumissions",
+                            "@inform_on_status": "mailto:dsde-ops@broadinstitute.org",
+                            "@inform_on_error": "mailto:dsde-ops@broadinstitute.org"
+                        }
+                    }
+                }
+            }
+        }
 
+        submission = submission_dict["SUBMISSION_SET"]["SUBMISSION"]
         self.create_actions(submission)
         self.create_submission_attributes(submission)
 
-        validate_xml(root, SUBMISSION_XSD)
-        write_xml_file("submission.xml", root)
+        validate_xml(submission_dict, SUBMISSION_XSD)
+        write_xml_file("submission.xml", submission_dict)
 
 
 # Helper Functions #
-def validate_xml(xml_tree, xsd_url):
+def validate_xml(xml_dict, xsd_url):
     try:
+        # Convert dictionary to XML string
+        xml_string = xmltodict.unparse(xml_dict)
+        
         # Download XSD content
         xsd_content = requests.get(xsd_url).content
         # Create XMLSchema object
         xmlschema_doc = etree.parse(BytesIO(xsd_content))
         xmlschema = etree.XMLSchema(xmlschema_doc)
-        # Convert ElementTree to string and parse
-        xml_string = ET.tostring(xml_tree)
-        xml_doc = etree.fromstring(xml_string)
+        # Parse the XML string (convert to bytes first)
+        xml_doc = etree.fromstring(xml_string.encode())
         # Validate XML against XSD
         xmlschema.assertValid(xml_doc)
         print("Validation successful. XML is valid according to XSD.")
```

Comment:
For logging/debugging, might be helpful to change `XSD` to the schema that was used